packages/backend/code/index.ts

```diff
@@ -53,6 +53,29 @@ const endTurn = (roomCode: string): void => {
   updateGameForRoom(rooms[roomCode]);
 };
 
+/**
+ * Randomize teams in a room.
+ * @param room
+ */
+const randomizeTeams = (room: Room): void => {
+  let players: PlayerData[] = room.players;
+  players = shuffle(players);
```

Comment:
Lodash `shuffle` is not a mutating function, it returns a new shuffled array. Doing `players = shuffle(players)` will set the local variable `players` to the new array, but it does not change the array stored in `room.players`. Recommend saving directly to `room.players` as shown below.

Client doesn't have access to the `room` object, so we can pass the `roomCode` here instead.

```suggestion
/**
 * Randomize teams in a room.
 * @param roomCode
 */
const randomizeTeams = (roomCode: string): void => {
  const room = rooms[roomCode];
  room.players = shuffle(room.players);
```

Reply:
Ah, yeah. Makes sense it works like that. I thought it was like that initially, but when I was testing it since it still "randomized" the teams because the initial color was random I thought it was randomizing them in the room as well oops. Also true the client should be able to access the function on its own with the code.